### ISSUE TITLE
object `try` can't work with objects that don't decorate anything

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/eo#1433]
This test:
```
[] > try-inf
  QQ.hamcrest.assert-that > @
    try
      []
        QQ.math.positive-infinity > @
      [e]
        e > @
      nop
    $.equal-to QQ.math.positive-infinity
```

fails with:   
Errors: [ERROR]  EOtry_infTest.testWorks:64->PhDefault.attr:256->PhDefault.attr:256->PhDefault.attr:256->PhDefault.attr:255 » Ex EOorg.EOeolang.EOstringν183="The object is at the line 68, position 8; caused by Error at "description-of.σ" attribute; caused by Error at "equal-to.x" attribute; caused by The object is at the line 158, position 15; caused by ExUnset: Error at "EOorg.EOeolang.EOmath.EOpositive_infinity#Δ" attribute; caused by ExUnset: Can't get(), attribute "Δ" is absent among other 20 attrs (signum, ρ, minus, σ, mod, as-int, lt, eq, gt, plus, as-string, div, neg, times, abs, as-bytes, gte, pow, neq, lte) and φ is absent"SF

`positive-infinity` does not decorate anything (it does not have `> @` inside). I think in that way we may expect `try` should return just `positive-infinity`

<img width="668" alt="image" src="https://user-images.githubusercontent.com/37025995/200945783-3088f2ec-d153-42c2-902b-f0d0e35b4655.png">

**Comment by mximp** [comment]
@maxonfjvipon With `try` object we expect an exception to be raised during some process. And this process is dataization of the body of try (which is 1st argument). 
In your case `positive-infinity` object is not dataizable (i.e. doesn't have `Δ` attribute set) hence it's dataized into an `ExUnset` object (i.e. EO error) for attribute `Δ`.
It would be misleading to make `try` to return its body in case it cannot be dataized.

The code above can be changed as follows:
```
[] > try-inf
  QQ.hamcrest.assert-that > @
    try
      seq
        QQ.math.positive-infinity
        TRUE
      [e]
        FALSE > @
      nop
    $.equal-to FALSE
```

**Comment by maxonfjvipon** [comment]
Solved here: #1792
@mximp Thanks
